orchagent/portsorch.cpp

```diff
@@ -1311,6 +1437,20 @@ void PortsOrch::doPortTask(Consumer &consumer)
                     }
 
                 }
+
+                if (pfc_asym != "")
+                {
+                    if (setPortPfcAsym(p, pfc_asym))
+                    {
```

Comment:
Consider to move "m_portList[port.m_alias] = port;" within setPortPfcAsym out of the function to here. As setPortPfcAsym should focus on set port asym pfc mode.

Reply:
"m_portList[port.m_alias] = port" is inside "setPortPfcAsym" intentionally.
It is because before updating "asym_mode" we need to get current value (line 546) and only then update.